javascript/src/stories/pages/About.stories.js

```diff
@@ -1,12 +1,12 @@
-import React from 'react';
+// import React from 'react';
 
-import About from "main/pages/About/About";
+// import About from "main/pages/About/About";
 
-export default {
-  title: 'pages/About/About',
-  component: About
-};
+// export default {
+//   title: 'pages/About/About',
+//   component: About
+// };
 
-const Template = () => <About />;
+// const Template = () => <About />;
 
-export const Empty = Template.bind({});
+// export const Empty = Template.bind({});
```

Comment:
If you are removing this story, please delete the file instead of commenting out the lines.

Reply:
Bryan may have marked this as resolved, but I need to see this file deleted before I'd consider this mergable.  

It is not acceptable professional practice to check in "commented out" code to the main branch.